src/main/resources/static/js/main.js

```diff
@@ -150,47 +149,65 @@
         latitude: position.coords.latitude,
         longitude: position.coords.longitude
       });
+      geometry=true;
       locationButton.toggleClass('loading');
     }, function () {
       locationButton.toggleClass('loading');
       locationButton.toggleClass('disabled');
     })
   });
 
-  const geocodingButton = $('.ui.button#geocoding'); // it is not used
   const card = $('#pac-card');
   const input = document.getElementById('pac-input');
```

Comment:
You can use the jQuery syntax all the way

Reply:
I wish but I can't make it work. In the previous commit, I already tried it as

```
#1
const input = $('#pac-input'); //OR const input = $('input#pac-input')
#2
const infowindowContent = $('#infowindow-content');
```
Using #1 OR/AND  #2 the JS console shows the following errors:

For #1 - autocomplete does not work and it can be seen on the JS console

```
InvalidValueError: not an instance of HTMLInputElement
```

For #2  - autocomplete works but it can be seen on the JS console

```
InvalidValueError: setContent: not a string; and [object Object]
```

I searched for those traces (#1 and #2) in StackOverflow like

# 1 - https://stackoverflow.com/a/34064547/2684377 --> It has to do with Google maps autocomplete now supports only window.HTMLInputElement (INPUT tag). So I tried by `const input = $('input#pac-input')` but does not work either 
# 2 - I have not found anything specifing to autocomplete

I have tried several suggestions but none of them worked

That's why I kept the classic JS way... in the end, Jquery is a layer in the top on JS to make things easier...